### ISSUE TITLE
For swin-B training in pad prompt

### ISSUE THREAD

**Issue by DasMadhureeta** [KMnP/vpt#60]
To train the Swin-B with padded prompt, what will the image size to use as 224X224 showing an error for sliding window? Can you please help me with what image size to use while adding a padded prompt as the learning rate and weight decay value?
For cifar-10 and cifar-100 dataset.
